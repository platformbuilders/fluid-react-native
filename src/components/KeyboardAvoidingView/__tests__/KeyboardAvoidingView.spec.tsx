import React, { View } from 'react-native';
import renderer from 'react-test-renderer';
import Keyboard from '..';

describe('<Keyboard />', () => {
  it('should render Keyboard', () => {
    const wrapper = renderer.create(
      <Keyboard>
        <View />
      </Keyboard>,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
