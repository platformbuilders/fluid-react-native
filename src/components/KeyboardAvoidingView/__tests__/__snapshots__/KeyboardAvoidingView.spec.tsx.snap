// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Keyboard /> should render Keyboard 1`] = `
<View
  onLayout={[Function]}
  style={
    {
      "flex": 1,
    }
  }
>
  <View />
</View>
`;
